workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,200 @@
+
+![](https://lh7-us.googleusercontent.com/nAcWz6onmEDjro8MHLJW0QuAMxlrLqaoBsR--nCRpyRXrXOMnC6TItKPMU0JFcc8sUlhyYTqqtxFGCzwSuNuxDzlUrqfAephKP5kf_GXSKt9IwQ1ljclEQUhbnbw_Hov-Cvhu0B38ZbwBj1tU4U0Hes)
+
+![](https://lh7-us.googleusercontent.com/vnHYyys7-lF4JJ0iWNe1U_9tzKBaugYaHecw6-jK-QcyReb5eurxA7mRonIGf9mYI6KO0cVcbRB5i4WPy2DkILnpSglgxYzPllWNoUa-bX5YpWtvdY2M1rRDoaEaTQE7qwyqqkDRET3fX1bJsjoS-fY)
+
+You have the option to create a dashboard from an existing data source or create a widget for an existing dashboard. The following describes how to create a dashboard from an existing data source. For more information on creating a widget, navigate to the [Create New Widget](/workflow/custom-dashboard/dashboard-builder-overview#create-new-widget) section.
+
+Create a new dashboard by selecting a data source to start your dashboard with:
+
+![](https://lh7-us.googleusercontent.com/awzI7eStfpqKWy78cOJdW-8pMyrVL6lrocPdQnmVlqrEtGh6QFjABrm-GoaCyG3URvGjdRnm_Ukzw4ZrbY6U-Li9QlY-eWKJ3dFzTzO_fb6IT48onLDEUZxaJ5EPDKJwWm6WLs_hzjmGJJTsCvq6AgQ)
+
+Select the metrics you would like to include as widgets in your dashboard:
+
+![](https://lh7-us.googleusercontent.com/MYBkwh9gWohgGbuF9CScQlUi5RU-0w0J8AVXTY3_MqEnidFnqTzGgXSLMy4iVcpV3_aRgrA3fCnxRZU53vGuL5Lp1RUL7ljVXmRdJfhbqptdDmcATY9LDiTDG715qWoX5ABfvTFuFVgmqArf6UC1i1I)
+
+Once you have added widgets to your dashboard, you will be taken directly to your dashboard:
+
+<Note> Data will appear momentarily after you have created your new dashboard. </Note>
+
+![](https://lh7-us.googleusercontent.com/rg4LfJetovyC5UMrgE7H3aN3UUuK0K2kbbHp5LYgGHLs12M-48jW3OtN7RlPA0sJnTznOpcdtNFqghehlroWY7lhsw1eQsE1ZSKXSqf8e-EUNX99RzcCr24ScSxFiXhp27NvLokBGwV9PMuKWk8jwTY)
+
+# Create New Widget
+
+Creating a new widget is useful for custom reporting and exploration. To create a new widget, navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
+
+![](https://lh7-us.googleusercontent.com/Qn2pnY24-2CuG7moVbGCi4UqO0aoIMdqNjsYaSipLHo3gfUOhD_z0vchbVyMNrN3zb6gPZeCAkkoJWmndzgqtcZVVFz9jzJYD_gkhP3qWNytupLefOGdMGLhd5Z9SjTvxqXbMmM1x6XYF0QzXK9LNn8)
+
+![](https://lh7-us.googleusercontent.com/Oe2a3IYXnQ4JTYXRbzMe9c7RZSHXFySFLJGFZCxWhJIFIvlNM2hOmGgQdCapl2h8Hk5-FhEWRwALQxZ1lJ2BGJtnvQnXA_r0TCqX5o3OeXwJnAqQrR1wkm77-uZLmL5V7QTiGpPr4Uhc6SRvjaO3Yjg)
+
+### Chart Types
+
+Choose from one of the following types of charts for your widget:
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+| __Chart Type__     | __Use Case__ |
+| :-------------     | :------------|
+| __Timeseries__     | Describes trends that are collected and ordered with respect to time  |
+| __Bar__            | Helpful for visually comparing differences categorical data and groups |
+| __List__           | Ideal for comparing numerical data associated with numerical data |
```

Comment:
I think numerical data got duplicated here twice? Maybe "Ideal for  comparing numerical data in combination with multiple attributes"